docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+This management module configures HTTP & HTTPS proxy settings.
```

Comment:
We want to focus on the user, not the product. So, please rewrite this sentence as "You can ... do something in the location."

Reply:
You can configure HTTP & HTTPS proxy settings with this management module.